### ISSUE TITLE
Characteristics of Intended Reader structure should be consistent between examples

### ISSUE THREAD

**Issue by bmaclach** [JacquesCarette/Drasil#1129]
As discussed during the Feb. 15th, 2019 meeting, the characteristics of the intended reader should be divided into 3 categories:
- Assumed knowledge
- Knowledge related to the topic
- Knowledge that would be an asset

As of PR #1128, the function for building the Characteristics of the Intended Reader section of the SRS accepts 4 parameters:
- A parameter for topics the reader "should have a strong knowledge in"
- A parameter for topics the reader "should have an understanding of"
- A parameter for standards the reader "should be familiar with"
- A parameter for topics that "would be an asset" if the reader understood

Each case study should recharacterize the content of this section into the 3 new categories so that we can have a consistent way of building this section between examples.

**Comment by smiths** [comment]
This sounds good to me.

**Comment by JacquesCarette** [comment]
Good design - and implementation.  Is there an outstanding 'to do' here? PR has not yet been merged (changes needed), but it will be.  It seems to me that the last line in the issue calls for a 'to do list' that needs to be checked off once each case study has been taken care of.

**Comment by bmaclach** [comment]
Yes, this still needs to be done for all of the case studies:

- [ ] Gamephys
- [ ] GlassBR
- [ ] NoPCM
- [ ] SSP
- [ ] SWHS

I wasn't sure if I was supposed to do this for just SSP or for all of the case studies? (I'm fine to do it for all of them, just didn't want to step on any toes if that's not what was expected). Also, I'm not fully sure about how to distinguish between assumed knowledge and knowledge related to the topic. As an example, with SSP, "solid mechanics" seems definitely topic-related, but would "level 4 physics" be assumed or topic-related?

**Comment by JacquesCarette** [comment]
If it's an easy 1-2 line change (easy to find, easy to fix), go ahead. Otherwise it should be assigned to whoever owns that case study.  Whenever you are not sure, let the owner handle it.

**Comment by bmaclach** [comment]
It should be an easy to find and fix change, and I think it makes more sense for a single person to own this, so I'll keep it assigned to myself. The tricky part is just going to be deciding on how to categorize each bit of knowledge as assumed, topic-based, or asset.

**Comment by JacquesCarette** [comment]
Tricky == guess but verify.

**Comment by bmaclach** [comment]
I guessed, which ended up just meaning that I categorized everything as "topic-based" and left the "assumed" knowledge empty for each example (see PR #1155). I think this makes sense because I think the nature of "assumed" knowledge means that we were unlikely to have included them in this section prior to now.

I'm not sure how best to verify my guesses. Is @smiths the right person to look them over? He is assigned as a reviewer on the PR.
